### ISSUE TITLE
[1.12.2] Custom themes don't work

### ISSUE THREAD

**Issue by The-Key** [TeamJM/journeymap#5]
#### JourneyMap Version:
1.10.2-5.5.2
1.12.1-5.5.0
1.12.2-5.5.2
#### Forge Version:
1.10.2 - 12.18.3.2185
1.12.2 - 14.23.0.2491
#### Describe the problem in repeatable steps:
1. Copied "flat\Stronghold.theme2.json" to "flat\Keytheme.theme2.json"
2. Edited first 5 lines of "flat\Keytheme.theme2.json" to:
```json
{
  "schema": 2,
  "author": "key676",
  "name": "KeyTheme",
  "directory": "flat",
```
3. Started Minecraft and created new superflat, creative world.
4. Opened fullscreen map and clicked theme button until it cycled back to OceanMonument.
5. KeyTheme never appeared.
#### Relevant errors from `.minecraft/journeymap/journeymap.log`:
Probably these?
```
[23:27:08] [main/ERROR] [JMLogger] Default theme not found in C:\Program Files (x86)\MultiMC\instances\SNJ RotP\minecraft\journeymap\icon\theme\flat\OceanMonument.theme2.json: OceanMonument (SUPPRESSED)
```
```
[23:27:09] [main/ERROR] [FileHandler] Couldn't locate icons for Default: C:\Program%20Files%20(x86)\MultiMC\instances\SNJ%20RotP\minecraft\mods\journeymap-1.12.2-5.5.2.jar\assets\journeymap\icon\entity
```
[journeymap.log](https://github.com/TeamJM/journeymap/files/1484385/journeymap.log)

**Comment by techbrew-mc** [comment]
Fixed in 5.6.0b1

**Comment by brnnnrsmssn** [comment]
i must be doing something wrong.

in journeymap 1.12.2-5.5.3 which i just downloaded, after copying the purist theme and editing the name, author, and filename, i save then launch minecraft. open fullscreen map and cycle through themes my theme does not appear.

filename: motionlessAccord.theme2.json
author: b33f3r
name: motionlessAccord

the file is placed in the .minecraft/journeymap/icon/theme/flat folder with all the other themes.
